pontoon/base/migrations/0040_remove_changed_entity_locale_entries_for_repository_projects.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.15 on 2023-02-23 16:02
+from django.db import migrations
+from pontoon.base.models import Project, Resource, Entity, ChangedEntityLocale
+
+
+def remove_changed_entity_locale_entries_for_repository_projects(apps, schema_editor):
+    ChangedEntityLocale.objects.filter(
+        entity__resource__project__data_source=Project.DataSource.DATABASE
+    ).delete()
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ("base", "0039_mark_system_users"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=remove_changed_entity_locale_entries_for_repository_projects,
```

Comment:
Sorry just realized that we should add a reverse_code here, [noop](https://docs.djangoproject.com/en/4.1/ref/migration-operations/#django.db.migrations.operations.RunPython.noop) is fine.

See migration [`0023_fuzzy_to_missing.py`](https://github.com/mozilla/pontoon/blob/1ae0a58ad0537eb35d80393187202fd523df8bec/pontoon/base/migrations/0027_keep_pretranslation_enabled.py) for example.